### PR TITLE
Better Dev Docs

### PR DESCRIPTION
**Pull request by KenwoodFox**
To help with onboarding, me and @Kredcool are working on better docs for new code members to get started.

### CODE REVIEW

File: docs/developing/developing.rst [hunk 1]
```diff
@@ -56,20 +56,56 @@ Requirements
...
+
+under the ``dashboard`` directory run
+
+.. code-block:: shell
+
+   pipenv install # May take a while
+   pipenv shell
+   make run # makes a local hosted website dashboard
+
+This will start the website dashboard. See :ref:`Using the Web Station` for more info.
```
Comment (KenwoodFox) [inline docs/developing/developing.rst]: Nice! You catch on quick!

File: docs/developing/developing.rst [hunk 2]
```diff
@@ -56,20 +56,56 @@ Requirements
...
+   cd www && python -m pynetworktables2js
+   16:32:28:683 INFO    : dashboard           : Connecting to NetworkTables at 127.0.0.1
+   16:32:28:683 INFO    : nt                  : NetworkTables initialized in client mode
+   16:32:28:683 INFO    : dashboard           : NetworkTables Initialized
+   16:32:28:684 INFO    : dashboard           : Listening on http://localhost:8888/
+   16:32:28:978 INFO    : tornado.access      : 101 GET /networktables/ws (127.0.0.1) 0.89ms
+   16:32:28:978 INFO    : net2js              : NetworkTables websocket opened
+
+Within this you will see one called local host, you will need to copy that link and put it in a web browser.
+It is recommened that you use this in incognito due to network cache causing issues with not updating.
```
Comment (KenwoodFox) [inline docs/developing/developing.rst:186]: Great!

File: docs/developing/developing.rst [hunk 3]
```diff
@@ -98,11 +98,11 @@ Activating the dashboard doesn't automaticaly start. To start the Web Station wi
 .. code-block::
 
    cd www && python -m pynetworktables2js
-   16:32:28:683 INFO    : dashboard           : Connecting to NetworkTables at 127.0.0.1
```
Comment (KenwoodFox) [inline docs/developing/developing.rst]: This is not necciary, localhost is 127.0.0.1

File: docs/developing/developing.rst [hunk 4]
```diff
@@ -98,11 +98,11 @@ Activating the dashboard doesn't automaticaly start. To start the Web Station wi
...
+   16:32:28:683 INFO    : dashboard           : Connecting to NetworkTables at Ip address
    16:32:28:683 INFO    : nt                  : NetworkTables initialized in client mode
    16:32:28:683 INFO    : dashboard           : NetworkTables Initialized
    16:32:28:684 INFO    : dashboard           : Listening on http://localhost:8888/
-   16:32:28:978 INFO    : tornado.access      : 101 GET /networktables/ws (127.0.0.1) 0.89ms
```
Comment (KenwoodFox) [inline docs/developing/developing.rst]: This is not necciary, localhost is 127.0.0.1

File: docs/introduction/introduction.rst [hunk 5]
```diff
@@ -4,4 +4,5 @@ Introduction
 .. warning::
    Make sure this manual's revision hash and the robot hash match!
 
-Introduction here.
+This is the code manual for team 1721 of First Robotics.
```
Comment (KenwoodFox) [inline docs/introduction/introduction.rst]: Great!

**Review by KenwoodFox — COMMENTED** [review]
Fix the `Ipaddress` section with `localhost` or `127.0.0.1`

**Review by KhanSimeoni — APPROVED** [review]
Looks good, but there are a few too many comments

**Pull request merged**